.travis.yml

```diff
@@ -96,6 +96,10 @@ install:
     - python setup.py install
     - python -c "import mne; mne.sys_info()"
     - python -c "import numpy;print(numpy.__version__)"
+    - python -c "import scipy;print(scipy.__version__)"
+    - python -c "import pandas;print(pandas.__version__)"
```

Comment:
not all travis builds have pandas - you get an error due to this line now. (the previous builds were all green)